platform/admin/core/admin-core-api/src/main/java/org/codice/ddf/ui/admin/api/ConfigurationAdminMBean.java

```diff
@@ -184,6 +184,17 @@ String createFactoryConfigurationForLocation(String factoryPid, String location)
     boolean update(String pid, Map<String, Object> configurationTable) throws IOException;
 
     /**
+     * Interceptor method for special handling of guest claims profile data.
+     *
+     * @param pid the persistent identifier of the configuration
+     * @param configurationTable the table of properties
```

Comment:
Currently out of scope since the standard `update(String, Map)` method does not specify either. Will add a note to `@see` the `GuestClaimsHandlerExt` object for more information.